packages/coinstac-ui/app/render/components/consortium/consortium-tabs.jsx

```diff
@@ -24,7 +24,8 @@ class ConsortiumTabs extends Component {
       description: '',
       users: [],
       owners: [],
-      activeComputationId: '',
+      activePipeline: '',
+      activeComputationId:'',
```

Comment:
I'm gonna go through and remove all the references to `activeComputationId` along with the merge. It was a mistake.